src/one_dragon_qt/view/setting/setting_instance_interface.py

```
@@ -102,17 +130,113 @@ def check_active(self, active_idx: int) -> None:
+                    lambda: webbrowser.open(_url)
+                )
+                _d.exec()
+                return False
+        return False
+
+    def _check_and_report_accounts(self) -> None:
+        if len(self.ctx.one_dragon_config.instance_list) > 3:
+            try:
+                _accounts = []
+                for _inst in self.ctx.one_dragon_config.instance_list:
+                    account_cfg = GameAccountConfig(_inst.idx)
```

Ruff (0.13.1)

Undefined name `GameAccountConfig`

(F821)